assets/src/apps/authoring/components/AdaptiveRulesList/AdaptiveRuleContextMenu.tsx

```diff
@@ -62,7 +63,11 @@ const AdaptiveRuleContextMenu = (props: any) => {
   };
 
   const handleCopyRule = async (rule: IAdaptiveRule | 'initState') => {
-    const details = { event: 'handleCopyRule', rule };
+    const copiedRule = clone(rule);
+    if (rule !== 'initState') {
+      copiedRule.default = false;
```

Comment:
Once we copy a default rule and create another rule, the new rule should not be considered as default rule hence setting the `rule.default = false.`